pkg/cloudprovider/aws/block_store.go

```diff
@@ -102,6 +125,7 @@ func (b *BlockStore) CreateVolumeFromSnapshot(snapshotID, volumeType, volumeAZ s
 		SnapshotId:       &snapshotID,
 		AvailabilityZone: &volumeAZ,
 		VolumeType:       &volumeType,
+		Encrypted:        &encrypted,
```

Comment:
Sorry I didn't notice this on the first pass, but we don't even need the `isSnapshotEncrypted` helper anymore since we're already calling `DescribeSnapshots` just above to get the tags anyway.  This whole PR can be collapsed into editing this line to say:

```go
Encrypted: snapRes.Snapshots[0].Encrypted,
```

Reply:
OK, updated to include new change.